### PR TITLE
Update os_image documentation

### PR DESCRIPTION
**Pull request by akshay1610**
+label: docsite_pr

##### ISSUE TYPE
- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by mnaser** [comment]
shipit

File: lib/ansible/modules/cloud/openstack/os_image.py [hunk 1]
```diff
@@ -91,6 +91,8 @@
       username: admin
       password: passme
       project_name: admin
+      os_user_domain_name: Default
```
Comment (Akasurde) [inline lib/ansible/modules/cloud/openstack/os_image.py:94]: Can we add a note [here](https://github.com/ansible/ansible/blob/cc8e90395a9892eb58efac02ae1ac9737853d694/lib/ansible/utils/module_docs_fragments/openstack.py#L34) stating these parameters are also available ? Thanks.

**Review by Akasurde — DISMISSED** [review]

File: lib/ansible/modules/cloud/openstack/os_image.py (hunk 1 above)
Comment (gundalow) [reply lib/ansible/modules/cloud/openstack/os_image.py:94]: @anotherjesse Hi, would you be able to address this comment?

Comment (acozine) [reply lib/ansible/modules/cloud/openstack/os_image.py:94]: @gundalow I think you had an auto-complete fail . . . @akshay1610 could you address @Akasurde's comment?

Comment (acozine) [reply lib/ansible/modules/cloud/openstack/os_image.py:94]: Addressed by 6488201bf9d64575a31fb3cca9c86fad41eba361

**Pull request merged**